test/commands/optimize.test.ts

```diff
@@ -84,17 +84,26 @@ describe('optimize', () => {
         expect(ctx.stderr).toContain('ValidationError');
         done();
       });
+  });
 
+  describe('with no context file', () => {
+    beforeEach(() => {
+      try {
+        testHelper.deleteDummyContextFile();
+      } catch (e) {
+        if (e.code !== 'ENOENT') {
+          throw e;
+        }
+      }
```

Comment:
You can create helper (separate function with this body) for that in the `test/testHelper.ts`

Reply:
Yeah, I was considering that option but finally decided to repeat this piece of code in all those specific tests. Basically, we were in trouble with `deleteDummyContextFile()` since it was called twice for the tests checking that the context file does not exist, and the 2nd call was failing due to the context file had been already deleted. As all the logic was encapsulated in the `testHelper.ts` it was difficult to identify where was exactly happening the problem in the specific tests. Besides, I had to simplify the logic of `deleteDummyContextFile()` cause it was failing randomly due to conflicts related to the management of the execution for the different threads.

Apart from that, I don't think it's a good idea to include that 'try-catch' block in the helper since the errors would be "hidden" in that case and not spotted, so that contributors would end up making use just of the function with the 'try-catch' and ignoring the original 'deleteDummyContextFile()' function, I guess 🤔 

Let's say I'd prefer not to follow DRY principles when it comes to unit tests, something like the following approach: https://dev.to/mbarzeev/why-practicing-dry-in-tests-is-bad-for-you-j7f